src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTraceHelper.cs

```diff
@@ -33,7 +33,17 @@ public static bool IsTraceEnabled()
             return s_TraceEnabled;
         }
 
-        public static void TraceCallbackStatus(string message, IntPtr handle, bool invokeCallback, uint status)
+        public static void Trace(string format, params object[] args)
```

Comment:
> I think for now I'll just revise the method to have explicit arguments

Sounds good (noting of course that if you make those separate args typed as object, you'll get boxing for any value types you pass).

> I didn't realize that using params object[] would cause that extra allocation

Yeah, the compiler needs to allocate the object[] in which to pass the values.  Roslyn has an optimization that'll let it use ```Array.Empty<object>()``` instead of allocating an array, but that only applies for the case where you don't pass any args.